### DISCUSSION TITLE
Hugo-Theme-Jane  V3 Upgrade Notice !!!

### DISCUSSION THREAD

**Discussion by xianmin** [xianmin/hugo-theme-jane#397] [Announcements]
Hugo-theme-jane just had a major update, please check out: [Jane Theme V3 Major Update](https://www.xianmin.org/hugo-theme-jane/post/jane-theme-v3-major-update/) .

I would like to have **5 real-world examples** using the new version of Hugo-Theme-Jane V3, and include them in the `README` . It would be ideal if the sites are in different languages from various countries, with distinct color schemes, and preferably not newly created websites. If you're interested, feel free to leave a message here!

Any suggestions are welcome. Thank you for liking this theme.

**Comment by crazyoptimist** [comment]
Thanks for your hard work!

I've just upgraded my blog to V3.
https://crazyoptimist.net/

Compared to your blog, which is in Chinese, mine doesn't look the best.

I really liked the original card design from the older version (I mean, padding, font sizes, etc. Simply put, I'm not a fan of the underline of titles, and the separator, narrow padding inside the card.). And, wrapping the header contents in a container would be better IMHO. One more thing, google search is broken?

By the way, the reading experience has definitely improved in V3, I mean, for individual posts. I'm also impressed by the card content update, which displays all styles and code blocks as they are.

Cheers!

**Reply by xianmin** [reply]
Just saw your message, thanks for letting us know you're using this theme!

Yes, the card design on the homepage is not perfect. Because I personally want a new more personalized homepage like the recent update https://www.xianmin.org/hugo-theme-jane/ . The original design I need to take time to improve again.

Yes, google search needs to be fixed.

If you have any suggestions for a better design, feel free to raise an issue.

**Reply by crazyoptimist** [reply]
I got it. FYI, social links are broken also.

**Reply by xianmin** [reply]
It's been fixed.

**Reply by crazyoptimist** [reply]
Thanks!

**Reply by crazyoptimist** [reply]
Found another UI glitch. On pad screens, responsiveness is broken.

![2024-11-04_03-39](https://github.com/user-attachments/assets/c4beaf83-e264-4889-9682-651980c31026)
![2024-11-04_03-40](https://github.com/user-attachments/assets/29083a52-4f70-417f-9162-9e69089dedff)
![2024-11-04_03-42](https://github.com/user-attachments/assets/3bc70031-adde-433b-aa2e-e60f4a7f1cc0)

**Reply by xianmin** [reply]
I will check later.

**Reply by xianmin** [reply]
fixed, you can update the theme and see the effect.

**Reply by crazyoptimist** [reply]
Great! I've verified that it's been fixed.

Since my header height is different from the default one, I had to add the below custom styling:

```scss
.desktop-header {
  height: 74px;
}

.desktop-navbar {
  height: 100%;
  .menu {
    .menu-item {
      margin: auto;
      height: 100%;
    }
  }
}
```

Before your fix, the second block was not needed. :thinking:

**Reply by xianmin** [reply]
https://github.com/xianmin/hugo-theme-jane/commit/6e237ed462b298e36053dfd21da9062eafe43d28

Well, I think we could add a scss setting to modify the site header height.

Now, you can modify the `_settings.scss` value to change the header height:

```scss
$desktop-header-height: 4rem;
$mobile-header-height: 3rem;
```
